components/Editor.js

```diff
@@ -1179,10 +1179,70 @@ Prism.languages.clike = {
     Prism.languages.markup.tag.addInlined('script', 'javascript'),
   (Prism.languages.js = Prism.languages.javascript);
 
+/* "prismjs/components/prism-reason" */
```

Comment:
This is code from a separate PR. Happy to merge in with this if you address the comment from the other PR; use an Object not Map.